src/importers/azureDevops.js

```diff
@@ -63,9 +69,27 @@ export async function getOrganizationInfo() {
   return result;
 }
 
-export async function getWorkItems(organization, offset) {
+export async function getProjectInfo(organization) {
+  if (!organization) {
+    return []
+  }
+
+  let endPoint = `https://dev.azure.com/${organization}/_apis/projects?api-version=6.0`;
+  let forWhat = "projectInfo";
+  const result = await sendGetHttpRequest(endPoint, forWhat);
+
+  return result;
+}
+
+export async function getWorkItems(organization, project, offset) {
   const body = {
-    query: "Select  [System.Id], [System.Title], [System.State] From WorkItems",
+    query: `
+      Select  [System.Id], [System.Title], [System.State]
+      From WorkItems
+      Where [State] <> 'Done'
+      And [System.TeamProject] = '${project}'
+      Order by [Microsoft.VSTS.Common.Priority] asc, [System.CreatedDate] desc
```

Comment:
The square brackets are giving me flashbacks to SQL Server/MS Access from 21 years ago…

Reply:
Microsoft Access! I haven't thought about that in a long time. 😆